### DISCUSSION TITLE
Keep up the good work

### DISCUSSION THREAD

**Discussion by rtcw-nihi** [rtcwmp-com/rtcwPro#429] [General]
I apologize for being MIA. I am so happy to see the progress that has been made and that this is helping keep RTCW alive <3

One day, I will return.

**Comment by wolf6542** [comment]
Rascal.

**Comment by krazykaze81** [comment]
Rtcw 4 life ❤️

**Comment by rtcw-nihi** [comment]
Nearly a year later, and I am still MIA :( It is great to see the incredible (continued) progress! Well done! Much love to y'all and to the entire community! Like I said, one day, I will return :) Keep up the good work!

**Reply by rtcw-nihi** [reply]
Please share this with Snappas, as I can't edit the participants.......

**Reply by snappas** [reply]
The github events show up on discord :). Appreciate the kind words nihi. Keeping it moving forward is the goal.

**Reply by krazykaze81** [reply]
Nihi, do you have a way we can contact you in case web server or domain stops working?
